sonar-project.properties

```diff
@@ -1,3 +1,2 @@
 sonar.javascript.exclusions=**/jest.config.js,**/__mocks__/**,**/node_modules/**,**/test/**,**/test-output/**
 sonar.javascript.lcov.reportPaths=test-output/lcov.info
-sonar.sources=app
```

Comment:
`sonar.sources` was added to `sonar-project.properties` in an earlier failed attempt to configure SonarCloud. This was the wrong file to edit so shouldn't have been left in that PR.